lessons/03-html-lists/index.html

```diff
@@ -41,39 +41,37 @@ <h1>Notable Teachers and Staff Members</h1>
                         <li>Rowena Ravenclaw</li>
                         <li>Salazar Slytherin</li>
                 </ol>
-                 <h2>List Of Spells</h2>
-               
-                 
-                            <dt>Aberto </dt>
-                            <dd>A spell presumably used to open objects such as doors or windows.</dd>
-                        </dl> 
-                        <dl>
-                                <dt>Accio</dt>
-                                <dd>This charm summons an object to the caster, potentially over a significant distance.</dd>
-                            </dl> 
-                            <dl>
-                                    <dt>Expecto Patronum </dt>
-                                    <dd>Evokes a partially-tangible positive energy force known as a Patronus.</dd>
-                                </dl> 
-                                <dl>
-                                        <dt>Sectumsempra </dt>
-                                        <dd> A dark spell that creates large, blood-oozing gashes on the subject as if said subject had been "slashed by a sword".</dd>
-                                    </dl> 
-                                    <dl>
-                                            <dt>Immobulus </dt>
-                                            <dd>Renders living targets immobile..</dd>
-                                        </dl> 
-                                        <dl>
-                                                <dt>Partis Temporus </dt>
-                                                <dd>Creates a temporary gap through protective magical barriers.</dd>
-                                            </dl> 
-                                            <dl>
-                                                    <dt> Locomotor </dt>
-                                                    <dd>The spell causes the named object to rise in the air and move around at the will of the caster.</dd>
-                                                </dl> 
-                
-                 
-        
         </ul>
+        <h2>List Of Spells</h2>
+                
+       
+        <dl>        
+        <dt>Aberto </dt>
+        <dd>A spell presumably used to open objects such as doors or windows.</dd>
+        </dl> 
+        <dl>
+                <dt>Accio</dt>
+                <dd>This charm summons an object to the caster, potentially over a significant distance.</dd>
+        </dl> 
+        <dl>
+                <dt>Expecto Patronum </dt>
+                <dd>Evokes a partially-tangible positive energy force known as a Patronus.</dd>
+        </dl> 
```

Comment:
With datalists you can actually have many `<dt>` and `<dd>`'s per `<dl>`, so this whole thing could be refactored into:

```html
<dl>        
  <dt>Aberto </dt>
  <dd>A spell presumably used to open objects such as doors or windows.</dd>
  <dt>Accio</dt>
  <dd>This charm summons an object to the caster, potentially over a significant distance.</dd>
  <dt>Expecto Patronum </dt>
  <dd>Evokes a partially-tangible positive energy force known as a Patronus.</dd>
  <!-- etc... -->
</dl> 
```

Reply:
OH GOT IT....thanks,would make ammends asap